### PR TITLE
Added group `log`

### PR DESCRIPTION
**Pull request by RichardLitt**
This is dependent on https://github.com/ipfs/go-ipfs/issues/2305 being merged, in case subsystem is not an enumerated type list, and in case all of the possible types are not mentioned.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -991,9 +991,247 @@ Outputs the list of peers that were removed.
...
+
++ Parameters
+    + arg1 (string) - The subsystem logging identifier. Use `all` for all subsystems.
+        + Members
+            - bitstwap
+            - blockstore
+            - dht
+            - merkledag
+            - all
+    + arg2 (enum[string])
```
Comment (dignifiedquire) [inline apiary.apib]: Missing description of `arg2`

File: apiary.apib [hunk 2]
```diff
@@ -991,9 +991,247 @@ Outputs the list of peers that were removed.
...
+        - Code: 0
+
+    + Body
+
+        ```
+        {
+          "Message": "Error: No such logger",
+          "Code": 0
+        }
+        ```
```
Comment (dignifiedquire) [inline apiary.apib:2562]: I think apiary will be fine if you leave out the body if all attributes have examples

File: apiary.apib [hunk 3]
```diff
@@ -991,9 +991,247 @@ Outputs the list of peers that were removed.
...
+        Content-Type: application/json
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        Date: Fri, 05 Feb 2016 21:22:27 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (Error)
+        - Message: "Error: No such logger"
```
Comment (dignifiedquire) [inline apiary.apib]: Missing types?

File: apiary.apib [hunk 4]
```diff
@@ -991,9 +991,247 @@ Outputs the list of peers that were removed.
...
+        - event (string)
+        - peerID (Multihash)
+        - session (string)
+        - system (string)
+        - time (string)
+
+    + Body
+
+        ```
+        {"event":"updatePeer","peerID":"QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ","session":"90950f01-a408-455a-a560-767f326b7331","system":"dht","time":"2016-02-05T21:36:44.191536188Z"}
```
Comment (dignifiedquire) [inline apiary.apib]: Please add line breaks

File: apiary.apib (hunk 1 above)
Comment (RichardLitt) [inline apiary.apib]: There is no description for arg2 in the CLI. Thoughts?

File: apiary.apib (hunk 2 above)
Comment (RichardLitt) [inline apiary.apib:2562]: Sure. But I won't be. :) Going to keep this in.

File: apiary.apib (hunk 3 above)
Comment (RichardLitt) [inline apiary.apib]: Nah: That it is an Error type means I don't need to specify types for these. The Type specification covers that.

File: apiary.apib (hunk 1 above)
Comment (dignifiedquire) [inline apiary.apib]: Maybe sth like this `- The log level, with`debug`the most verbose and`panic`the least verbose.`
Also are these two args optional or required, if they are optional what is the default value :P

File: apiary.apib (hunk 2 above)
Comment (dignifiedquire) [inline apiary.apib:2562]: ok :D

File: apiary.apib (hunk 3 above)
Comment (dignifiedquire) [inline apiary.apib]: :+1:

**Comment by RichardLitt** [comment]
Waiting on #2362.

File: apiary.apib (hunk 1 above)
Comment (RichardLitt) [inline apiary.apib]: Required.

**Comment by RichardLitt** [comment]
I think this is done now. RFM?

**Comment by dignifiedquire** [comment]
LGTM

**Pull request merged**